dashboard/lib/policies/races.rb

```diff
@@ -39,9 +39,9 @@ class Policies::Races
   def self.any_urm?(races)
     races = normalize(races)
 
-    return nil unless races
```

Comment:
The value returned by this helper method is persisted into the user model here: https://github.com/code-dot-org/code-dot-org/blob/c2cbc28460d454f135ed36bc89d8e0e7b1fff51c/dashboard/app/models/user.rb#L675

That value is then referenced in a number of places, including serialization of user data. We also have some logic [here](https://github.com/code-dot-org/code-dot-org/blob/c2cbc28460d454f135ed36bc89d8e0e7b1fff51c/dashboard/app/models/user.rb#L2463) which indicates we consider `nil` to be a valid value for this property.

All of which is to say: it is not clear to me whether the change from `nil` to `false` in this context is safe.